### PR TITLE
Reorganise front matter of API changelog

### PR DESCRIPTION
**Pull request by fmcgrath**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,16 @@
...
-| [SA108](mapping/sa108_mapping_v1.csv)<br />Capital Gains Summary | [individuals-capital-gains-income-api](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/individuals-capital-gains-income-api) |
+Many parameters in some Income Tax (Making Tax Digital) APIs map to box numbers in <a href="https://www.gov.uk/self-assessment-tax-return-forms">Self Assessment tax return forms (GOV.UK)</a>. For more information, see [Mappping CSV files](mapping/mapping-csv-files.md).
```
Comment (ben-HMRC) [inline README.md:9]: Maybe just "Parameters in some Income Tax"... etc

Comment (ben-HMRC) [inline README.md:9]: Mappping -> Mapping

File: README.md [hunk 2]
```diff
@@ -1,70 +1,16 @@
...
-2. Click the "Watch" drop-down at the top right, and choose "Custom.. Releases"
-3. Go to: https://github.com/settings/notifications
-4. Under "Subscriptions.. Participating, @mentions and custom", ensure that you have "Email" ticked
-
-You should now receive an email approximately every two weeks containing the changes made to the APIs since the last update.
-
-## Roadmap
-
-For planned changes, see the [Income Tax (Making Tax Digital) roadmap](https://developer.service.hmrc.gov.uk/roadmaps/mtd-itsa-vendors-roadmap/).
+This page contains a log of the latest changes for [Income Tax (Making Tax Digital) APIs](https://developer.service.hmrc.gov.uk/api-documentation/docs/api?filter=income-tax-mtd). For information about planned changes to Income Tax (Making Tax Digital) APIs, see [Income Tax (Making Tax Digital) roadmap](https://developer.service.hmrc.gov.uk/roadmaps/mtd-itsa-vendors-roadmap/).
```
Comment (ben-HMRC) [inline README.md:3]: Could avoid repeating "Income Tax (Making Tax Digital) APIs" so many times by changing to "For information about planned changes to these APIs, see [our roadmap]."

**Review by ben-HMRC — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (fmcgrath) [reply README.md:9]: OK. Although Dale and Rajar spent a lot of time considering that sentence.

**Pull request merged**